src/test/java/org/lisoft/lsml/model/item/AmmoWeaponTest.java

```diff
@@ -104,4 +104,15 @@ public final void testSpreadQuirks() throws Exception {
 
         assertEquals(normal * 2, quirked, 0.0);
     }
+    
+    @Test
+    public void testGetVolleySize() throws Exception {
+        final AmmoWeapon ac20 = (AmmoWeapon) ItemDB.lookup("AC/20");
+        final AmmoWeapon lrm10 = (AmmoWeapon) ItemDB.lookup("LRM 10");
+        final AmmoWeapon clrm10 = (AmmoWeapon) ItemDB.lookup("C-LRM 10");
+        
+        assertEquals(10, lrm10.getVolleySize());
+        assertEquals(1, clrm10.getVolleySize());
+        assertEquals(1, ac20.getVolleySize());
+    }
 }
```

Comment:
Parsing was my worry.  I can add some getRawFiringPeriod in the mix. I did not add UACs or RACs, but otherwise, done.

Reply:
Testing parsing is all good, just be mindful that actual values may be changed by PGI in a patch and that could break tests and that causes maintenance work for us. I don't think these particular values will change, but then again I've been wrong on that before. Some times it's better to test for "non default value" rather than specific value. But I think these particular tests will be fine.